tests/run_smoke_test.sh

```diff
@@ -19,7 +19,7 @@ test -f $TEST_FUNCTIONS_FILE
 
 echo "[*] Running Weaver with $TEST_FUNCTIONS_FILE"
 
-./bin/weaver -f $TEST_FUNCTIONS_FILE ./bin/tester > $OUTPUT_FILE&
+./bin/weaver --json -f $TEST_FUNCTIONS_FILE ./bin/tester > $OUTPUT_FILE&
```

Comment:
Nice catch, I committed before without even running my own shitty test :)

With that in mind, this change will invalidate the `tester_output_sorted_slurped.json` file anyway. I'll write a script for updating that file probably tomorrow.

Reply:
ooo or a github action if it'll let me load eBPF